SWLOR.Game.Server/Feature/AbilityDefinition/Force/ThrowRockAbilityDefinition.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System.Collections.Generic;
+using SWLOR.Game.Server.Core;
+using SWLOR.Game.Server.Core.NWScript.Enum;
+using SWLOR.Game.Server.Core.NWScript.Enum.VisualEffect;
+using SWLOR.Game.Server.Service;
+using SWLOR.Game.Server.Service.AbilityService;
+using SWLOR.Game.Server.Service.CombatService;
+using SWLOR.Game.Server.Service.PerkService;
+using SWLOR.Game.Server.Service.SkillService;
+
+namespace SWLOR.Game.Server.Feature.AbilityDefinition.Force
+{
+    public class ThrowRockAbilityDefinition : IAbilityListDefinition
+    {
+        public Dictionary<FeatType, AbilityDetail> BuildAbilities()
+        {
+            var builder = new AbilityBuilder();
+            ThrowRock1(builder);
+            ThrowRock2(builder);
+            ThrowRock3(builder);
+            ThrowRock4(builder);
+            ThrowRock5(builder);
+
+            return builder.Build();
+        }
+
+        private static void ImpactAction(uint activator, uint target, int level, Location targetLocation)
+        {
+            var dmg = 0;
+
+            switch (level)
+            {
+                case 1:
+                    dmg = 10;
+                    break;
+                case 2:
+                    dmg = 15;
+                    break;
+                case 3:
+                    dmg = 25;
+                    break;
+                case 4:
+                    dmg = 34;
+                    break;
+                case 5:
+                    dmg = 43;
+                    break;
+            }
+
+            dmg += Combat.GetAbilityDamageBonus(activator, SkillType.Force);
+
+            var attackerStat = GetAbilityScore(activator, AbilityType.Willpower);
+            var defense = Stat.GetDefense(target, CombatDamageType.Force, AbilityType.Willpower);
+            var defenderStat = GetAbilityScore(target, AbilityType.Willpower);
```

Comment:
Per Discord convo, let's have this target VIT instead of WIL on the defender's side, to differentiate it from the other damage abilities slightly.